frontend/src/pages/RecipientSearch/components/RecipientResults.js

```diff
@@ -85,8 +85,8 @@ export default function RecipientResults(
         <thead>
           <tr>
             {renderColumnHeader('Region', 'regionId')}
-            {renderColumnHeader('Recipient Name', 'name')}
-            {renderColumnHeader('Program Specialist', 'programSpecialist')}
+            {renderColumnHeader('Recipient name', 'name')}
+            {renderColumnHeader('Program specialist', 'programSpecialist')}
```

Comment:
I think this change is wrong as these are proper names of titles and not sentence text.

Reply:
Confirmed with Adam Roux that these are supposed to be sentence text